packages/next-swc/crates/next-core/js/tsconfig.json

```diff
@@ -4,37 +4,50 @@
     "strict": true,
     "noFallthroughCasesInSwitch": true,
     "skipLibCheck": true,
-
     // interop constraints
     "allowSyntheticDefaultImports": true,
     "esModuleInterop": true,
-
     // js support
     "allowJs": true,
     "checkJs": false,
-
     // environment
     "jsx": "react-jsx",
-    "lib": ["ESNext", "DOM"],
+    "lib": [
+      "ESNext",
+      "DOM"
+    ],
     "target": "esnext",
-
     // modules
     "baseUrl": ".",
-    "module": "esnext",
+    "module": "Node16",
```

Comment:
See: https://devblogs.microsoft.com/typescript/announcing-typescript-5-2/#module-and-moduleresolution-must-match-under-recent-node-js-settings

Reply:
Should we use lowercase to match the `moduleResolution` below so they are exactly the same?